config/routes/api.rb

```diff
@@ -79,6 +84,16 @@
       end
     end
 
+    unless Rails.env.production?
```

Comment:
Are you disabling v1 in prod here? I see that above as well. Can you extract to a method or feature toggle?
```
def api_v1_enabled?
   !Rails.env.production?
end
```

Reply:
When I was putting this together I wasn't thinking we would immediately merge it and make these endpoints live. We might want to work on some more API controllers before doing more of a public "launch" for example, or we might spot some small changes we'd like to make whilst working on the _second_ or _third_ `v1` API controller, in which case we could adjust things a little bit without any problems and make them live at a slightly later date. 

That was roughly my thinking here :man_shrugging: